pyuvdata/uvdata/tests/test_miriad.py

```diff
@@ -262,7 +263,7 @@ def test_read_carma_miriad_write_ms(tmp_path):
     uv_in.history = uv_out.history
 
     # Final equality check
-    assert uv_in == uv_out
+    assert uv_in.__eq__(uv_out, allowed_failures=allowed_failures_with_ms)
```

Comment:
If you were to add the scan number to `utils`, I think you may be able to drop the added `allowed-failures` in the equality call. Not a strict requirement, but perhaps is a nicety.

Reply:
Done! I've reverted most of these extra `allowed_failures` to be only for `filename` as they originally were.